gtest/src/manager.rs

```diff
@@ -204,6 +204,34 @@ impl Program {
     }
 }
 
+/// Delayed dispatch with original message id and reservation info.
+#[derive(Debug)]
+pub(crate) struct DelayedDispatch {
+    pub message_id: MessageId,
+    pub dispatch: Dispatch,
+    pub reservation: Option<ReservationId>,
```

Comment:
why do we need this reservation? what's the aim of it?
delayed dispatches on pallet have existing gas node once created